worker/instancemutater/mutater.go

```diff
@@ -87,6 +87,15 @@ func (m *mutater) startMachines(tags []names.MachineTag) error {
 				continue
 			}
 
+			profileChangeWatcher, err := api.WatchLXDProfileVerificationNeeded()
```

Comment:
This introduces a 2nd methodology for what to ignore in terms of WatchLXDProfileVerificationNeeded, which I believe will be confusing in the long term.  In the case of KVM containers we don't call WatchLXDProfileVerification, then in the case of manual machines, we fail WatchLXDProfileVerification.

Suggest `if api.IsManual() { continue}` before the call to api.WatchLXDProfileVerificationNeeded() as an alternative.  

This keeps more business logic of the instance mutater in the worker rather than spreading more out to different packages.